grype/matcher/rpmdb/matcher_mocks_test.go

```diff
@@ -58,3 +59,11 @@ func (pr *mockProvider) GetByDistro(d *distro.Distro, p pkg.Package) ([]vulnerab
 
 	return pr.data[ty+":"+d.FullVersion()][p.Name], nil
 }
+
+func (pr *mockProvider) GetByCPE(request syftPkg.CPE) (v []vulnerability.Vulnerability, err error) {
```

Comment:
Added because we're now testing the Public `Match` method being exposed by this package.